Markdown/Default.sublime-keymap

```diff
@@ -0,0 +1,61 @@
+[
+    // Auto-pair asterisks
+    { "keys": ["*"], "command": "insert_snippet", "args": {"contents": "*$0*"}, "context":
+        [
+            { "key": "setting.auto_match_enabled"},
+            { "key": "selector", "operand": "text.html.markdown - markup.raw" },
+            { "key": "selection_empty", "match_all": true },
+            { "key": "preceding_text", "operator": "regex_contains", "operand": "\\S" },
```

Comment:
We should add a condition that prevents it from triggering when preceded by a backslash, since backslashes are used to escape markup characters. Same for `` ` ``.

Reply:
Great point. Fixed.